### ISSUE TITLE
Mobile: Sideways  scrolling and zoom in and out are  enabled in mobile mobile devices.

### ISSUE THREAD

**Issue by ermatitan** [titan-cebu/Residential-Building-Brokers#9]
![image](https://cloud.githubusercontent.com/assets/12791909/11432754/9cd7e878-94ef-11e5-98d9-c6571c4e7662.png)

**Comment by titanguest** [comment]
This is per Bernard's request, as it is need to be raised to Jolene for confirmation.

**Comment by ermatitan** [comment]
A confirmation is needed before we can close the issue. Thanks

**Comment by ermatitan** [comment]
Won't fix issue as confirmed by Bernard Salas
